narayana-release-process.sh

```diff
@@ -62,6 +62,12 @@ then
   exit
 fi
 
+# check if docker is actually podman
+podman --version
```

Comment:
Please can you add something like https://github.com/jbosstm/narayana/blob/f3a0bbb5784b21d1d097311d1e15b4028b64679d/narayana-release-process.sh#L16 to check the command is there

Reply:
@tomjenkinson we don't require podman. We only need a different flag when its being used. Or is this only about `command -v`?

Reply:
Ah, I think I understand better now. If you are trying to verify if podman exists then I think using `command -v podman` would be good instead of running the command. Thanks